### DISCUSSION TITLE
Move governance items like process out of charter

### DISCUSSION THREAD

**Discussion by taylor** [lfn-cnti/bestpractices#127] [Ideas]
We could make the charter the items we see not changing for a much longer period and move other items out.  Examples
- Current chairs
- Roles
- Election process

Example of a splitting them out: https://github.com/cncf/sig-security/tree/master/governance

Reference @iawells comments from https://github.com/cncf/cnf-wg/pull/122#issuecomment-812068086
> I am not going to object to doing it this way, but I wonder if adding this in the charter makes sense (we're not changing the charter when the co-chairs change), and it would additionally mean we can't be inconsistent if it's only in one place.

and @xmulligan response:
>@iawells I agree. I think the Governance should be pulled out of the charter as a separate file similar to the contributors.md. That would also solve your issue

**Comment by xmulligan** [comment]
I created a PR for this https://github.com/cncf/cnf-wg/pull/129
